src/lib/connectors/repos/queries/gauging-stations.js

```diff
@@ -0,0 +1,15 @@
+exports.findGaugingStations = `select 
+water.licence_version_purpose_condition_types.*,
+water.licence_version_purpose_conditions.*,
+water.licence_version_purposes.*,
+water.licence_versions.end_date,
+water.gauging_stations.* from 
+water.gauging_stations join water.gauging_station_condition on (water.gauging_stations.gauging_station_id = water.gauging_station_condition.gauging_station_id)  
```

Comment:
This table doesn't exist `gauging_station_condition`;

I think you meant to join on `water.licence_gauging_stations`

Reply:
Yes, water.licence_gauging_stations is a new table described in https://eaflood.atlassian.net/browse/WATER-3109 (renamed)